sdk/cmake/msvc.cmake

```diff
@@ -29,7 +29,15 @@ endif()
 
 add_definitions(/Dinline=__inline /D__STDC__=1)
 
-add_compile_flags("/X /GR- /EHs-c- /GS- /Zl /W3")
+if(NOT USE_CLANG_CL)
+    add_compile_flags("/X /Zl")
```

Comment:
`/X` is pretty important. If it's not supported, should we pass it `-nostdinc` or something instead?